src/dali_executor/dali_pipeline.h

```diff
@@ -54,6 +54,10 @@ class DaliPipeline {
     if (this != &dp) {
       ReleasePipeline();
       ReleaseStream();
+      serialized_pipeline_ = std::move(dp.serialized_pipeline_);
```

Comment:
Please double check, if moving like this requires assigning default value `dp.serialized_pipeline_={}`

Reply:
It doesn't